sqldb-dynamodb/README.md

```diff
@@ -0,0 +1,18 @@
+# sqldb-dynamodb capability provider
+
+This capability provider allows wasmCloud actors to use AWS DynamoDB
+through PartiQL, and implements the "wasmcloud:sqldb" capability contract.
+
+Build with 'make'. Test with 'make test'.
+
+### Capabilities:
+- execute Parti statements (select, insert, update, etc.)
+- query and execute only differ in that query will return the result
+
```

Comment:
Would love to have a link definition values section here since there's a lot of configuration that you can possibly use, e.g. https://github.com/wasmCloud/capability-providers/tree/main/kv-vault#link-definition-configuration-settings

Reply:
It's actually the same as blobstore. I'll copy from there.